pcs/cli/constraint/output/colocation.py

```diff
@@ -154,8 +159,16 @@ def plain_constraint_to_cmd(
         warn(
             "Resource instance(s) detected in constraint "
             f"'{constraint_dto.attributes.constraint_id}' but not supported by "
-            "this command."
+            "this command"
+        )
+        return []
+    if constraint_dto.node_attribute is not None:
+        warn(
+            "Option 'node_attribute' detected in constraint "
+            f"'{constraint_dto.attributes.constraint_id}' but not supported by "
+            "this command"
```

Comment:
warning should be more specific about not including the affected constraint in generated commands.